dataset_generation/wikisplit/readme.md

```diff
@@ -0,0 +1,22 @@
+# Generation of a dataset from wikipedia revision history
```

Comment:
Sorry my bad. can we leave the file name for readme as README.md only?

Reply:
yes, ill change that.